src/app/add/add-idea/idea.component.scss

```diff
@@ -37,7 +37,8 @@
     padding: 3rem;
     background-color: white;
 
-    width: 40rem;
+    max-width:100%;
+    max-height:700px;
```

Comment:
@richa208 Please use `rem` as the preferred unit.

1rem has been set 10px as default.

Reply:
done